rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -369,7 +370,13 @@ def _decode_normal_claim(self, context: DecoderContext) -> DecodingOutput:
         raw_amount = hex_or_bytes_to_int(context.tx_log.data[128:160])
         amount = asset_normalized_value(amount=raw_amount, asset=self.vcow_token)
         airdrop_identifier: Literal['cow_mainnet', 'cow_gnosis'] = 'cow_mainnet' if self.evm_inquirer.chain_id == ChainID.ETHEREUM else 'cow_gnosis'  # noqa: E501
+        claim_has_payment = hex_or_bytes_to_int(context.tx_log.data[32:64]) in (1, 2)
```

Comment:
add a comment explaining what 1 and 2 mean. You or anyone in the team will come in the future and will need to research it